src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -92,16 +92,21 @@ private async Task<int> VerifySignatureForPackageAsync(string packagePath, ILogg
 
                 var packageIdentity = packageReader.GetIdentity();
 
-                logger.LogInformation(Environment.NewLine + string.Format(CultureInfo.CurrentCulture,
+                logger.LogMinimal(Environment.NewLine + string.Format(CultureInfo.CurrentCulture,
                     Strings.VerifyCommand_VerifyingPackage,
                     packageIdentity.ToString()));
                 logger.LogInformation($"{packagePath}{Environment.NewLine}");
 
                 var logMessages = verificationResult.Results.SelectMany(p => p.Issues).ToList();
-                await logger.LogMessagesAsync(logMessages);
+
+                //log issues first
+                IEnumerable<SignatureLog> issues = logMessages.Where(m => m.Level < LogLevel.Warning);
```

Comment:
nit: 
Multiple assignments to the same var can sometimes be error prone when coding. 

You can consider directlly calling logger.LogMessagesAsync on the expression.

Reply:
Addressed this feedback in the latest commit.